src/main/kotlin/com/workos/common/http/PaginationParams.kt

```diff
@@ -11,6 +11,7 @@ open class PaginationParams @JvmOverloads constructor(
   after: String? = null,
   before: String? = null,
   limit: Int? = null,
+  order: String? = null,
```

Comment:
Is there a way for us to type this as `desc` | `asc` ?

Reply:
Made order enum instead